docs/reference/versions.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Versions"
+layout: default
+sort: 8
+---
+
+# Versions and deprecation
+{: .no_toc}
+
+---
+
+## Supported versions
+
+Node Feature Discovery follows [semantic versioning](https://semver.org/) where
+the version number consists of three components, i.e. **MAJOR.MINOR.PATCH**.
+
+The most recent two minor releases (or release branches) of Node Feature
+Discovery are supported. That is, with X being the latest release, **X** and **X-1**
+are supported and **X-1** reaches end-of-life when **X+1** is released.
+
+## Deprecation policy
+
+### Feature labels
+
+Built-in [feature labels](../usage/features.md) and
+[features](../usage/customization-guide.html#available-features) are supported
+for 2 releases after being deprecated, at minimum. That is, if a feature label
+is deprecated in version **X**, it will be supported in **X+1** and **X+2** and
+may be dropped in **X+3**.
+
+### Configuration options
+
+Command-line flags and configuration file options are supported for 1 more
```

Comment:
any relations to helm values / operator (CR) values ?

Reply:
Good point @adrianchiris.

Regarding Helm charts '#1283

Regarding the operator we need to handle that separately because it's maintained in a separate repo:
https://github.com/kubernetes-sigs/node-feature-discovery-operator/issues/198